README.md

```diff
@@ -2167,7 +2177,8 @@ Currently being reworked as Neo Launcher.
 
 <img alt="GreenStash️Icon" height="64" src="https://raw.githubusercontent.com/Pool-Of-Tears/GreenStash/main/fastlane/metadata/android/en-US/images/icon.png">
 
-- [ ] Google Play
+- [x] [Google Play](https://play.google.com/store/apps/details?id=com.starry.greenstash)
```

Comment:
Especially that I understand that maintaining and developing software needs money because it's your free time and volunteering. Thank you very much! :)

Reply:
Thanks! The Google Play version doesn't contain any trackers or extra functionality compared to F-Droid or GitHub. Its sole purpose is to serve as an alternative platform for donations for users who would like to support my work. This helps me to cover associated costs such as hosting the API server and domain renewals, without me having to spend money out of my own pocket :)